json/Explain_Element_UserSkill.txt

```diff
@@ -3124,7 +3124,7 @@
 	{
 		"assign": "446",
 		"jp_text": "Ｓ１：賭死の撃",
-		"tr_text": "S1:Risky Strike",
+		"tr_text": "S1:Death Pool Strike",
```

Comment:
https://github.com/Arks-Layer/PSO2ENPatchCSV/pull/745/files

So this should be "S1:Reckless Strike" now?

Reply:
Yeah i guess so